### PR TITLE
Added documentation on how to use the package

### PR DESCRIPTION
**Pull request by mattBrzezinski**
Note: Targetting `MB/init` as I had based off of this branch. Change this before merging to `master`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
 # AWSClusterManagersDemo.jl
-Demo for AWSClusterManagers.jl presented at JuliaCon2021
+## Overview
+This demo project was presented at JuliaCon2021 to showcase a minimal example of the [AWSClusterManagers.jl](https://github.com/JuliaCloud/AWSClusterManagers.jl) package on AWS Batch. The job submitted to Batch calculates the approximate value of PI. The explanation for this can be found in this [article](https://jccraig.medium.com/calculate-pi-with-a-dartboard-bdb433f1c999).
+
+The job will:
+1. Launch four worker nodes
+2. Have each worker run a small process
+3. Retrieve and combine the results
+4. Display the results to CloudWatch logs
```
Comment (mjram0s) [inline README.md:9]: Should you add instructions on how to view the results in CloudWatch logs?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+## Setup
+**Note: When running this package you will be responsible for all AWS charges in your account.**
+
+To launch this yourself:
+
+1. Store your AWS account ID in an variable
+   	`AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)`
+1. Create a new ECR (Elastic Cloud Registry) repository to host you Docker image as:
+   `aws cloudformation foobar`
```
Comment (mjram0s) [inline README.md]: Should this be something like `aws cloudformation deploy --template-file ecr_template.yml`?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+2. Build the Docker image for this project
+   `docker build -t $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/awsclustermanagers-demo:latest .`
+3. Push the Docker image to the ECR repo created in the second step
+	`docker push $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/awsclustermanagers-demo:latest`
+1. Deploy the remainder of the Batch resources required to run the demo
+	`aws cloudformation foobar`
```
Comment (mjram0s) [inline README.md]: And here `aws cloudformation deploy --template-file batch_template.yml`?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,26 @@
...
+## Overview
+This demo project was presented at JuliaCon2021 to showcase a minimal example of the [AWSClusterManagers.jl](https://github.com/JuliaCloud/AWSClusterManagers.jl) package on AWS Batch. The job submitted to Batch calculates the approximate value of PI. The explanation for this can be found in this [article](https://jccraig.medium.com/calculate-pi-with-a-dartboard-bdb433f1c999).
+
+The job will:
+1. Launch four worker nodes
+2. Have each worker run a small process
+3. Retrieve and combine the results
+4. Display the results to CloudWatch logs
+## Setup
+**Note: When running this package you will be responsible for all AWS charges in your account.**
```
Comment (TylerLoewen) [inline README.md:11]: Would it be worth mentioning roughly how much this should cost to run so people are not scared away (if the cost is actually relatively low)?

File: README.md (hunk 2 above)
Comment (mattBrzezinski) [reply README.md]: Yes, whoops! Lol

File: README.md (hunk 4 above)
Comment (mattBrzezinski) [reply README.md:11]: Added this in, just placed a `$0.50 USD` placeholder, which is probably close enough to its actual cost.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,41 @@
...
+On-Demand instances at the time of writing are charged on a per-minute basis. An estimated cost for running the demo project and immediately destroying the stacks is around `$0.50 USD`.
+
+To launch this yourself:
+
+1. Store your AWS account ID in an variable
+   	`AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)`
```
Comment (mjram0s) [inline README.md]:
```suggestion
   	```
   	AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
   	```
```
Might be more readable if these were in code blocks

File: README.md [hunk 6]
```diff
@@ -1,2 +1,41 @@
...
+1. Create a new ECR (Elastic Cloud Registry) repository to host you Docker image as:
+    `aws cloudformation create-stack --template-body file://ecr_template.yml --stack-name awsclustermanagers-ecr`
+2. Build the Docker image for this project
```
Comment (mjram0s) [inline README.md]:
```suggestion
1. Build the Docker image for this project
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,41 @@
...
+    `docker build -t $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/awsclustermanagers-demo:latest .`
+3. Push the Docker image to the ECR repo created in the second step
```
Comment (mjram0s) [inline README.md]:
```suggestion
1. Push the Docker image to the ECR repo created in the second step
```

File: README.md (hunk 6 above)
Comment (mattBrzezinski) [reply README.md]: Markdown will actually number these appropriately for you! Make it easier for future maintenance so you don't need to re-number when inserting.

E.g.

```
1. Hello
1. World
1. Foo
1. Bar
```

Will turn into

```
1. Hello
2. World
3. Foo
4. Bar
```

Comment (mjram0s) [reply README.md]: I think you misread my suggestion. So right now your numbering goes:
1.
1.
2.
3.
1.
1.
3.

I was just changing everything to a 1 :p

Comment (mattBrzezinski) [reply README.md]: OOPS. Just typing too fast here. Just made the change and force pushed it.

**Review by mjram0s — APPROVED** [review]

**Pull request merged**